app/views/users/_user.html.erb

```diff
@@ -30,8 +30,10 @@
           followers
 
         <% end %>
-      </div>
 
+  
+      </div>
+    <% if policy(user).liked? %>
```

Comment:
I think this would be a different policy. Users should only see their own pending requests